script/pioneer/pioneer-qa.sh

```
@@ -0,0 +1,119 @@
+
+# Get the RPC_URL for the network ($ETH_NODE_URI_<NETWORK>)
+RPC_URL=$(eval echo \$ETH_NODE_URI_$UPPER_NETWORK)
+
+# Get from network id.
+FROM_NETWORK_ID=$(cast chain-id --rpc-url $RPC_URL)
+
+# Convert to_networks to array of ids.
+TO_NETWORKS_IDS=()
+for NET in $(echo $TO_NETWORKS | tr -d '[]"' | tr ',' '\n');
+do
+  NET_ID=$(cast chain-id --rpc-url $(eval echo \$ETH_NODE_URI_$(echo $NET | tr '[:lower:]' '[:upper:]')))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)